app/model/sdl/VehicleInfoModel.js

```diff
@@ -258,7 +258,50 @@ SDL.SDLVehicleInfoModel = Em.Object.create(
         'driverDoorAjar': true,
         'passengerDoorAjar': true,
         'rearLeftDoorAjar': true,
-        'rearRightDoorAjar': true
+        'rearRightDoorAjar': true,
+        'doorStatuses': [
```

Comment:
@LitvinenkoIra please declare door/gate/roof statuses as an empty arrays here. It will be overwritten by `populateModels()` method anyway as it is called on HMI start, so no need to hardcode so complex values here